qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -134,19 +134,60 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
             }
             hasNetworkViolation = results.any { isNetworkViolation(it) }
             hasProcessViolation = results.any { isProcessViolation(it) }
+            log.debug "Found violations in Splunk: \n${results}"
+            log.debug "hasNetworkViolation: ${hasNetworkViolation}\nhasProcessViolation: ${hasProcessViolation}"
             if (hasNetworkViolation && hasProcessViolation) {
-                log.info "Success!"
+                log.info "Success finding Network and Process Violations in Splunk!"
+                break
+            }
+        }
+
+        log.info "Starting conversion of ACS violations to Splunk alerts"
+        postToSplunk(port, "/services/saved/searches/" + SPLUNK_TA_CONVERSION_JOB_NAME +"/dispatch", [
+                "dispatch.now": "true",
+                "force_dispatch": "true",
+        ])
+
+        // Check for Alerts
+        List<Map<String, String>> alerts = Collections.emptyList()
+        boolean hasNetworkAlert = false
+        boolean hasProcessAlert = false
+        for (int i = 0; i < 20; i++) {
+            log.info "Attempt ${i} to get Alerts from Splunk"
+            // Hint: If this produces no results, evaluate expanding the earliest_time, e.g. to -15m.
+            // This can be done by expanding `createSearch` with a new parameter.
```

Comment:
Would this be something that would make sense for the test itself to do? Eg. retry with -15m on earliest_time if no results are produced? Under what circumstances would this happen, is it a rare edge case?

Reply:
I consider this the next course of action should the test still be flaky after these changes.
Ideally, we would never need to implement this. The test retries the current query over multiple minutes, so it should already catch all violations should there be any.